docs/prometheus-operator/README.md

```diff
@@ -0,0 +1,388 @@
+# Prometheus operator
+
+The purpose of this document is to explain how to setup [Prometheus operator](https://github.com/coreos/prometheus-operator) for Knative observability. It also describes how to scrape metrics endpoints using service monitors.
+
+## Cluster setup
+
+Create a kubernetes cluster as mentioned [here](https://knative.dev/docs/install/knative-with-gke/#creating-a-kubernetes-cluster).
+
+> Note: A GKE cluster is used, but any other cluster type can be used as well.
+
```

Comment:
Since the docs pointed here also contain instructions on installing knative, I feel it may be a little confusing. Maybe provide the simplest command to creating a GKE cluster and providing the cluster admin instructions.

Reply:
Done.